numpy/polynomial/hermite.py

```diff
@@ -77,7 +77,7 @@
 """
 import numpy as np
 import numpy.linalg as la
-from numpy.core.multiarray import normalize_axis_index
+from numpy._core.multiarray import normalize_axis_index
```

Comment:
This is publicly visible from `numpy.lib.array_utils`. I think it might be worthwhile to change internal imports from `_core` into imports from the public API. Not a blocker though. That said, doing this exercise might also be a good way to test your `ruff` PR.

Reply:
`ruff` would need to be able to suggest changing `numpy.core.absolute` to `numpy.absolute` but `numpy.core.multiarray.normalize_axis_index` to `numpy.lib.array_utils.normalize_axis_index`. There's probably lots of usages like this downstream so it would be nice if we could auto-fix them.

Reply:
I changed these imports to `numpy.lib.array_utils.*`.

Yes, I will include it in ruff rule.